packages/contracts-bedrock/periphery-deploy-config/optimism-goerli.json

```diff
@@ -14,5 +14,11 @@
   "faucetGelatoRecipient": "0x789e58a4B08A23a7f60141959C6ABbdC0D0C4Aba",
   "faucetGelatoBalanceV1DripInterval": 86400,
   "faucetGelatoBalanceV1Value": 1000000000000000000,
-  "faucetGelatoThreshold": 100000000000000000
+  "faucetGelatoThreshold": 100000000000000000,
+  "faucetOnchainAuthModuleAdmin": "0x8F0EBDaA1cF7106bE861753B0f9F5c0250fE0819",
+  "faucetOnchainAuthModuleAdminDripInterval": 86400,
+  "faucetOnchainAuthModuleAdminDripValue": 1000000000000000000,
```

Comment:
nit: These should follow the naming on the [module config struct](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/periphery/faucet/Faucet.sol#L42). Maybe `faucetOnchainAuthModuleTtl` and `faucetOnchainAuthModuleAmount`